opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -57,7 +59,7 @@ namespace Opm
         void init(const Wells* wells, const State& state, const PrevState& prevState)
         {
             // clear old name mapping
-            wellMap_.clear();
+            wellMap().clear();
```

Comment:
This should be unnecessary since it is cleared by BaseType::init(), if I read it correctly.